src/backend/TrafficCourts/TrafficCourts.Arc.Dispute.Service/Models/TcoDisputeTicket.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Newtonsoft.Json;
+using System.Text.Json.Serialization;
+
+namespace TrafficCourts.Arc.Dispute.Service.Models
+{
+    public class TcoDisputeTicket
+    {
+        [JsonProperty("citizen_name"), JsonRequired]
+        public string CitizenName { get; set; }
+        [JsonProperty("ticket_issuance_date"), JsonRequired]
+        public DateTime TicketIssuanceDate { get; set; }
+        [JsonProperty("ticket_file_number"), JsonRequired]
+        public string TicketFileNumber { get; set; }
+        [JsonProperty("issuing_organization"), JsonRequired]
+        public string IssuingOrganization { get; set; }
+        [JsonProperty("issuing_location"), JsonRequired]
+        public string IssuingLocation { get; set; }
+        [JsonProperty("drivers_license"), JsonRequired]
+        public string DriversLicense { get; set; }
+        [JsonProperty("ticket_counts"), JsonRequired]
+        public List<TicketDetails> TicketDetails { set; get; }
+        [JsonProperty("street_address")]
+        public string? StreetAddress { get; set; }
+        [JsonProperty("city")]
+        public string? City { get; set; }
+        [JsonProperty("province")]
+        public string? Province { get; set; }
+        [JsonProperty("postal_code")]
+        public string? PostalCode { get; set; }
+        [JsonProperty("email")]
+        public string? Email { get; set; }
+        [JsonProperty("dispute_counts")]
+        public Dictionary<string, DisputeDetails>[]? DisputeDetails { get; set; }
+    }
+
+    public class TicketDetails
+    {
+        public string section { get; set; }
+        public string subsection { get; set; }
```

Comment:
I think properties are CamelCase.
```suggestion
        public string Subsection { get; set; }
```